javascript/javascript3/week1/Weather/script.js

```diff
@@ -0,0 +1,44 @@
+//Weather API call
+const APIKEY = "9469946e53df5e8174588db17f99cd64";
+const limit = 3;
+
+const country = document.getElementById('search-country');
+const cityName = document.getElementById('search-city');
+const button = document.getElementById('button');
+
+const resultUl = document.getElementById('result-ul');
+
+button.addEventListener('click', (res)=>{
+    res.preventDefault();
+    let weatherCall = new Promise((resolve,reject) =>{
+    fetch(`http://api.openweathermap.org/geo/1.0/direct?q=${cityName.value},${country.value}&limit=${limit}&appid=${APIKEY}`)
+        .then(LatLonResponse => {
+            return LatLonResponse.json();
+        })
+        .then(LatLonJSON =>{
+            fetch(`http://api.openweathermap.org/data/2.5/weather?lat=${LatLonJSON[0].lat}&lon=${LatLonJSON[0].lon}&appid=${APIKEY}`)
+                .then(latlon =>{
+                    resolve(latlon.json());
+                }).catch(error =>{
+                    reject(error);
+            })
+        }).catch(error => {
+            reject(error);
+        });
+    })
+
+    weatherCall.then((message) =>{
```

Comment:
Nitpick(naming):
Maybe weather would be a better name if it comes from the result of the weatherCall?